nova/virt/vmwareapi/vmops.py

```diff
@@ -1537,18 +1538,16 @@ def _destroy_instance(self, context, instance, destroy_disks=True):
                                                                        vm_ref)
                 config_info_ex = cluster.propSet[0].val
 
-                if hasattr(config_info_ex, 'group'):
-                    for key, group in enumerate(config_info_ex.group):
-                        if not hasattr(group, 'vm'):
-                            continue
+            vm_groups = []
 
-                        for vm in group.vm:
-                            if vm.value == vm_ref.value and len(group.vm) == 1:
-                                cluster_util.delete_vm_group(
-                                            self._session, cluster.obj,
-                                            config_info_ex.group[key])
-                                break
-                        break
+            if hasattr(config_info_ex, 'group'):
+                for group in config_info_ex.group:
+                    if not hasattr(group, 'vm'):
+                        continue
+                    for vm in group.vm:
+                        if vm.value == vm_ref.value:
+                            vm_groups.append(group.name)
```

Comment:
Is this unused now that we do an updated query down at the end?

Reply:
Yes I'll remove what needed.